fmralign/pairwise_alignment.py

```diff
@@ -274,3 +275,26 @@ def fit_transform(self):
         raise AttributeError(
             "type object 'PairwiseAlignment' has no attribute 'fit_transform'"
         )
+
+    def get_parcellation(self):
+        """Get the parcellation masker used for alignment.
+
+        Returns
+        -------
+        labels: `list` of `int`
+            Labels of the parcellation masker.
+        parcellation_img: Niimg-like object
+            Parcellation image.
+        """
+        if hasattr(self, "pmasker"):
+            check_is_fitted(self)
+            labels = self.pmasker.get_labels()
```

Comment:
I really hat the `pmasker`. Can you change it to `parcel_masker` ?

Reply:
Sure!